code/modules/clothing/clothing.dm

```diff
@@ -106,8 +106,6 @@
 
 
 /obj/item/clothing/Destroy()
-	if(isliving(loc))

```

Comment:
Before anyone asks, this one is redundant, we deal with this on /item level anyway. If you know of any case where this is needed let me know.